lib/AST/AccessRequests.cpp

```diff
@@ -65,8 +65,8 @@ AccessLevelRequest::evaluate(Evaluator &evaluator, ValueDecl *D) const {
       // These are only needed to synthesize the setter.
       return AccessLevel::Private;
     case AccessorKind::Init:
-      // These are only called from designated initializers.
-      return AccessLevel::Private;
+      // These are only called from within the same module.
+      return std::min(storage->getFormalAccess(), AccessLevel::Internal);
```

Comment:
Why this change?

Reply:
The init accessor should be accessible within the module but it's currently impossible to use it across different files in the module.